showcase/src/assets/examples/table/with-footer/table.html

```diff
@@ -0,0 +1,3 @@
+<stark-table [data]="data" [columnProperties]="columns" [paginationConfig]="paginationConfig" [filter]="filter">
+	<header><h1 class="mb0" translate>Table with footer</h1></header>
```

Comment:
Remove the `translate` directive from this example ;)

Reply:
Done